src/app/components/turnos/dashboard/update-contacto-direccion.component.ts

```diff
@@ -107,11 +108,18 @@ export class UpdateContactoDireccionComponent implements OnInit {
         } else {
             this.arrayContactos = [this.contacto];
         }
-        if (this.paciente && this.paciente.direccion && this.paciente.direccion[0].valor) {
-            this.direccion = this.paciente.direccion[0];
-            if (this.direccion.ubicacion.provincia) {
+        if (this.paciente && this.paciente.direccion && this.paciente.direccion.length) {
+            let direccionOriginal = this.paciente.direccion[0];
+            if (direccionOriginal.valor) {
+                this.direccion.valor = direccionOriginal.valor;
+            }
+            if (direccionOriginal.ubicacion.provincia) {
```

Comment:
```suggestion
            if (direccionOriginal.ubicacion && direccionOriginal.ubicacion.provincia) {
```
`direccionOriginal.ubicacion` no podría ser `null`?